src/tracer.js

```diff
@@ -136,6 +136,17 @@ module.exports.restart = function restart() {
     tracerObj.trace.setToken(config.getConfig().token);
 };
 
+function stripOperations(traceJson) {
+    var filteredEvents = [];
+    traceJson.events.forEach(entry => {
```

Comment:
Instead of everything this might be neater:
`traceJson.events.filter(entry => entry.origin === 'runner' || entry.origin === 'trigger')`